src/tests/multihost/ipa/test_misc.py

```diff
@@ -557,6 +557,10 @@ def test_anonymous_pkinit_for_fast(self, multihost, backupsssdconf):
           5. Successfully get a ccache file with the FAST armor ticket
         :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1859751
         """
+        # skipping test for RHEL8.x as a bug BZ1859751 only valid for RHEL9
+        if '8.' in multihost.client[0].distro:
+            pytest.skip("Skipping test for RHEL8.x as automation of a bug BZ1859751 only valid for RHEL9")
```

Comment:
Why?

Reply:
Patches that fix #5961 were released upstream in 2.7.0 and are present both in RHEL9.1, RHEL8.7 and Fedora starting 35.

Reply:
Then @madhuriupadhye has to fix this test for RHEL8 as its failing :

https://ci-jenkins-csb-idmops.apps.ocp-c1.prod.psi.redhat.com/job/sssd-RHEL8.7/job/Gating-sssd/job/sssd-2.7.3-2.el8/job/pytest-sssd-ipa-tier1/1/consoleText

Reply:
@aborah-sudo tests are passing,
http://idm-artifacts.usersys.redhat.com/mupadhye/trigger//6619/test-suite//report.html
closing the MR.